pyocd/trace/swv.py

```diff
@@ -155,6 +157,10 @@ def run(self):
         if self._lock:
             self._lock.acquire()
 
+        tpiu_port = self._session.options.get('tpiu_port')
+        tpiu_server = StreamServer(tpiu_port, name="TPIU", is_read_only=True) \
+                      if tpiu_port else None
```

Comment:
The other servers in pyocd support setting the port to 0, which causes the OS to select any available high-numbered port. The actual port is then read back and printed in the "server started" log message. To do this for the raw SWV server might require adding a separate `swv_raw_server_enable`.